ttyd/umbrel-app.yml

```diff
@@ -0,0 +1,23 @@
+manifestVersion: 1
+id: ttyd
+category: DEVELOPMENT
+name: ttyd
+version: "1.7.3"
+tagline: Share your terminal over the web
+description: >-
+  ttyd is a simple command-line tool for sharing terminal over the web.
+releaseNotes: >-
+  Sixel support is off by default now, use -t enableSixel=true to enable it.
+developer: tsl0922
+website: https://tsl0922.github.io/ttyd/
+repo: https://github.com/tsl0922/ttyd
+support: https://github.com/tsl0922/ttyd/issues
+port: 7681
```

Comment:
I just checked and this port is already taken by [Lightning Shell](https://github.com/getumbrel/umbrel-apps/blob/master/lightning-shell/umbrel-app.yml). You can choose any free port for this (7682 is available for example). Do a global search of the umbrel-apps repo for `port: <your-port-choice>` and see if it is already taken in an apps `umbrel-app.yml`.